package.json

```diff
@@ -851,6 +851,11 @@
                             }
                         }
                     }
+                },
+                "appService.preDeployTask": {
```

Comment:
I don't see this used anywhere

Reply:
This call: https://github.com/Microsoft/vscode-azureappservice/pull/812/files#diff-2916a9eae992830600ca3418c2c75814R160 is actually a function in the shared package that leverages `preDeployTask` (linked below)

https://github.com/Microsoft/vscode-azuretools/pull/369/files#diff-4cd3198960c945ebbf0dea96b508d228R13